Mathlib/GroupTheory/Subsemigroup/Center.lean

```diff
@@ -23,92 +23,213 @@ import Mathlib.Data.Int.Cast.Lemmas
 
 We provide `Submonoid.center`, `AddSubmonoid.center`, `Subgroup.center`, `AddSubgroup.center`,
 `Subsemiring.center`, and `Subring.center` in other files.
+
+## References
+
+* [Cabrera García and Rodríguez Palacios, Non-associative normed algebras. Volume 1]
```

Comment:
Can you add this to references.bib?

Reply:
Oh nevermind, I somehow missed the line below.

Reply:
I added the missing part in after you commented :) https://github.com/leanprover-community/mathlib4/pull/6996/commits/c2f03f7eb24303e65848f842d0b8c49d1f6ce59b